evm/src/cpu/columns/mod.rs

```diff
@@ -2,14 +2,18 @@
 #![allow(dead_code)]
 
 use std::borrow::{Borrow, BorrowMut};
+use std::fmt::Debug;
 use std::mem::{size_of, transmute, transmute_copy, ManuallyDrop};
 use std::ops::{Index, IndexMut};
 
+use crate::cpu::columns::shared::CpuSharedColumnsView;
 use crate::memory;
 
+mod shared;
+
 #[repr(C)]
 #[derive(Eq, PartialEq, Debug)]
-pub struct CpuColumnsView<T> {
+pub struct CpuColumnsView<T: Copy> {
```

Comment:
I think we need this to deal with dropping `CpuGeneralColumnsView`, since it's union, unless we wrap stuff in `ManuallyDrop`.